From 88634f0a667a2a69b232f33a6f973ce443d476cc Mon Sep 17 00:00:00 2001
From: Boris Ranto <branto@redhat.com>
Date: Fri, 6 Nov 2015 12:15:36 +0100
Subject: [PATCH 03/11] ipmitool-1.8.11-ipmieved-pidfile.patch

596809: Initscript of the ipmievd service not LSB compliant

Don't print pidfile name to output by default.
And if verbose output is enabled, print the final pidfile name,
not the initial value.

Backported by: jsafrane
---
 src/ipmievd.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/ipmievd.c b/src/ipmievd.c
index f940579..218a4b6 100644
--- a/src/ipmievd.c
+++ b/src/ipmievd.c
@@ -703,7 +703,6 @@ ipmievd_main(struct ipmi_event_intf * eintf, int argc, char ** argv)
 
 	memset(pidfile, 0, 64);
 	sprintf(pidfile, "%s%d", DEFAULT_PIDFILE, eintf->intf->devnum);
-	lprintf(LOG_NOTICE, "ipmievd: using pidfile %s", pidfile);
 
 	for (i = 0; i < argc; i++) {
 		if (strncasecmp(argv[i], "help", 4) == 0) {
@@ -737,6 +736,7 @@ ipmievd_main(struct ipmi_event_intf * eintf, int argc, char ** argv)
 				__min(strlen((const char *)(argv[i]+8)), 63));
 		}
 	}
+	lprintf(LOG_DEBUG, "ipmievd: using pidfile %s", pidfile);
 
 	/*
 	 * We need to open interface before forking daemon
-- 
2.1.0

